### PR TITLE
Add workflow for fetching repo stats

### PR DESCRIPTION
**Pull request by louwers**

### CODE REVIEW

**Comment by HarelM** [comment]
I would advise to start with the readme and repo description.

**Comment by louwers** [comment]
@HarelM OK to include in this PR?

File: .github/workflows/store-repostats.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # The repositories to generate reports for
+        statsRepo: ['maplibre/maplibre-native', 'maplibre/maplibre-gl-native-distribution']
+      fail-fast: false
+      max-parallel: 1
+    steps:
+      - name: run-ghrs
+        uses: jgehrcke/github-repo-stats@RELEASE
```
Comment (HarelM) [inline .github/workflows/store-repostats.yml:21]: is the word "RELEASE" here correct?

File: .github/workflows/store-repostats.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          repository: ${{ matrix.statsRepo }}
+          ghtoken: ${{ secrets.ghrs_github_api_token }}
```
Comment (HarelM) [inline .github/workflows/store-repostats.yml:24]: Is this being saved somewhere? Should it deploy the stats to github pages?

**Comment by HarelM** [comment]
> OK to include in this PR?

Yes, this way I'll be able to review this better.

File: .github/workflows/store-repostats.yml (hunk 1 above)
Comment (louwers) [reply .github/workflows/store-repostats.yml:21]: Yes. This example is copied.

File: .github/workflows/store-repostats.yml (hunk 2 above)
Comment (louwers) [reply .github/workflows/store-repostats.yml:24]: Updated the README.md.

**Comment by louwers** [comment]
@HarelM I want to see if it works first, then I can share it on GitHub Pages.

The data is stored on a branch, I update the README.

**Review by HarelM — APPROVED** [review]

**Pull request merged**